### PR TITLE
More than one detector height range in ILLPowderDetectorScan

### PR DESCRIPTION
**Pull request by gui-co**
**Description of work.**

With this PR, it is now possible to provide more than one range for the `HeightRange` property of `PowderILLDetectorScan`. This was requested by our scientists to be able to run the data reduction on several ranges without needing to restart the full processing. The typical use case is the processing of the full detector and the central region.

Also, `InitialMask` and `FinalMask` properties have been removed as they were no longer needed.

The different ranges have to be provided as a list of comma separated values (example: "-0.05,0.05,-0.01,0.01" to process from -0.05m to 0.05m and from -0.01 to 0.01m). By default, the full detector is processed. If one or more ranges are provided, the full detector is always processed in addition to the specified regions.

**To test:**

Run `PowderILLDetectorScan` on some ILL/D2B systemtest data using the `HeightRange` property and observe the results:

```python
from mantid.simpleapi import PowderILLDetectorScan
PowderILLDetectorScan(Run="508093", HeightRange="-0.05,0.05,-0.01,0.01", OutputWorkspace="out")
```

With this script you should get 3 workspaces:
* `out_1D`: full detector
* `out_1D_-0.01, 0.01`: from -0.01m to 0.01m
* `out_1D_-0.05, 0.05`: from -0.05m to 0.05m

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.7.0/Diffraction/Powder/New_features/35284.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- It is now possible to provide more than one detector height range in :ref:`ILLPowderDetectorScan`
```
Comment (darominski) [inline docs/source/release/v6.7.0/Diffraction/Powder/New_features/35284.rst:1]: Could you also mention that two properties have been removed from this algorithm?

**Review by darominski — COMMENTED** [review]
Thank you for the more robust feature allowing to specify multiple height ranges. I have only one comment in this round of review, regarding the release note completeness.

File: docs/source/release/v6.7.0/Diffraction/Powder/New_features/35284.rst (hunk 1 above)
Comment (gui-co) [reply docs/source/release/v6.7.0/Diffraction/Powder/New_features/35284.rst:1]: Done

**Review by darominski — APPROVED** [review]
Thanks, looks good to me.

**Pull request merged**